catalog/samples/helloSwift3.swift

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Can we call this something else other than `helloSwift3`? I don't have a good suggestion yet, alas.